package.json

```diff
@@ -91,21 +91,21 @@
     "@angular-devkit/build-angular": "19.0.0-rc.1",
     "@angular-devkit/core": "19.0.0-rc.1",
     "@angular-devkit/schematics": "19.0.0-rc.1",
-    "@angular-eslint/builder": "18.0.1",
-    "@angular-eslint/eslint-plugin": "18.0.1",
-    "@angular-eslint/eslint-plugin-template": "18.0.1",
-    "@angular-eslint/schematics": "18.0.1",
-    "@angular-eslint/template-parser": "18.0.1",
-    "@angular-eslint/test-utils": "18.0.1",
+    "@angular-eslint/builder": "18.4.0",
```

Comment:
Updating NX **AND** @angular-eslint seems to do the trick.

The culprit was different NX versions, which both ran the post-install script. Sometimes this failed because one process would lock the other.

At https://github.com/ngrx/platform/actions/runs/11879387277 you can see 5 succeeding runs.